cypress/integration/channels.js

```diff
@@ -0,0 +1,134 @@
+import faker from "faker";
+
+import Channels from "../apiRequests/Channels";
+import { LEFT_MENU_SELECTORS } from "../elements/account/left-menu/left-menu-selectors";
+import { PRODUCTS_SELECTORS } from "../elements/catalog/product-selectors";
+import { ADD_CHANNEL_FORM_SELECTOS } from "../elements/channels/add-channel-form-selectors";
+import { CHANNEL_FORM_SELECTORS } from "../elements/channels/channel-form-selectors";
+import { CHANNELS_SELECTORS } from "../elements/channels/channels-selectors";
+import { CONFIGURATION_SELECTORS } from "../elements/configuration/configuration-selectors";
+import { HEADER_SELECTORS } from "../elements/header/header-selectors";
+import { DRAFT_ORDER_SELECTORS } from "../elements/orders/draft-order-selectors";
+import { ORDERS_SELECTORS } from "../elements/orders/orders-selectors";
+import { BUTTON_SELECTORS } from "../elements/shared/button-selectors";
+import { URL_LIST } from "../url/url-list";
+import ChannelsUtils from "../utils/channelsUtils";
+
+describe("Channels", () => {
+  const channelStartsWith = "Cypress:";
+  const currency = "PLN";
+  const channels = new Channels();
+  const channelsUtils = new ChannelsUtils();
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    channels.deleteTestChannels(channelStartsWith);
+  });
+
+  beforeEach(() => {
+    cy.clearSessionData().loginUserViaRequest();
+  });
+
+  it("should navigate to channels page", () => {
+    cy.visit("/")
+      .get(LEFT_MENU_SELECTORS.configuration)
+      .click()
+      .get(CONFIGURATION_SELECTORS.channels)
+      .click()
+      .location("pathname")
+      .should("contain", URL_LIST.channels);
+  });
+
+  it("should create new channel", () => {
+    const randomChannel = `${channelStartsWith} ${faker.random.number()}`;
+    cy.visit(URL_LIST.channels).waitForGraph("Channels");
+    channelsUtils.createChannelByView(randomChannel, currency);
+    cy.waitForGraph("Channel")
+      .get(ADD_CHANNEL_FORM_SELECTOS.backToChannelsList)
+      .click()
+      .get(CHANNELS_SELECTORS.channelsTable)
+      .contains(randomChannel)
+      .visit(URL_LIST.dashbord)
+      .get(HEADER_SELECTORS.channelSelect)
+      .click()
+      .get(HEADER_SELECTORS.channelSelectList)
+      .contains(randomChannel)
+      .click()
+      .visit(URL_LIST.products)
+      .waitForGraph("InitialProductFilterData");
+    cy.get(PRODUCTS_SELECTORS.productsList)
+      .first()
+      .click()
+      .get(PRODUCTS_SELECTORS.availableManageButton)
+      .click()
+      .get(PRODUCTS_SELECTORS.channelsAvailabilityForm)
+      .contains(randomChannel);
```

Comment:
Few comments would help maintain test in the future

```suggestion
    // New channel should be visible in channels list
    cy.waitForGraph("Channel")
      .get(ADD_CHANNEL_FORM_SELECTOS.backToChannelsList)
      .click()
      .get(CHANNELS_SELECTORS.channelsTable)
      .contains(randomChannel)
      
    // new channel should be visible in channel selector
    cy.visit(URL_LIST.dashbord)
      .get(HEADER_SELECTORS.channelSelect)
      .click()
      .get(HEADER_SELECTORS.channelSelectList)
      .contains(randomChannel)
      .click()
      
    // new channel should be visible at product availability form
    cy.visit(URL_LIST.products)
      .waitForGraph("InitialProductFilterData");
    cy.get(PRODUCTS_SELECTORS.productsList)
      .first()
      .click()
      .get(PRODUCTS_SELECTORS.availableManageButton)
      .click()
      .get(PRODUCTS_SELECTORS.channelsAvailabilityForm)
      .contains(randomChannel);
```